tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_unencapsulated.ts

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ViewEncapsulation,
+} from '@angular/core';
+@Component({
+  selector: 'line-chart-unencapsulated',
+  template: '',
+  styleUrls: ['line_chart_unencapsulated.css'],
```

Comment:
I think it would be worth moving the css out of the file and writing it as a string literal here. Then you have the entire hack in a single file and you can write one nice comment explaining the problem you're trying to solve and perhaps a little bit of detail about the hack.

Reply:
This does make sense. I actually considered doing that originally.